src/ghci/parse/ghc_message/exception.rs

```diff
@@ -0,0 +1,54 @@
+use winnow::PResult;
+use winnow::Parser;
+
+use crate::ghci::parse::lines::until_newline;
+
+use super::GhcMessage;
+
+/// Parse an "Exception" message like this:
+///
+/// ```plain
+/// *** Exception: /Users/.../dist-newstyle/ghc82733_tmp_1/ghc_tmp_34657.h: withFile: does not exist (No such file or directory)
+/// ```
+pub fn exception(input: &mut &str) -> PResult<GhcMessage> {
+    let _ = "*** Exception: ".parse_next(input)?;
+
+    let message = until_newline.parse_next(input)?;
```

Comment:
I think the most correct parser of a GHCi exception would parse until a prompt, not a newline. But until a newline would handle this particular exception, if that's all you're going for right now.

Reply:
Yeah, the issue is we get these in a stream of arbitrary GHC output, which may include other diagnostics we want to parse, compiling-such-and-such-a-module messages, etc.